File: Java/UberX.java
public class UberX extends Car{ // extends es para la herencia explicita
    private String brand;
    private String model;

    // CONSTRUCTOR
    public UberX(String license,Account driver,Integer passenger,String brand,String model){
        super(license, driver,passenger);
        this.brand = brand;
        this.model  = model;
    }

    // GETTERS AND SETTERS
    public String getBrand() {
        return brand;
    }

    public void setBrand(String brand) {
        this.brand = brand;
    }

    public String getModel() {
        return model;
    }

    public void setModel(String model) {
        this.model = model;
    }
    
    // METHODS AND ACTION 
    @Override
    public void printData(){
        System.out.println("Licencia: "+super.getLicense()+" Name Driver: "+super.getDriver()+" Pasajeros: "+super.getPassenger()+
        " Marca: "+getBrand()+" Modelo: "+getModel());
    }

}

File: PHP/UberX.php
<?php
    require_once('Car.php');
    
    class UberX extends Car{
        private $brand;
        private $model;

        // CONSTRUCTORES
        public function __construct($license,$driver,$brand,$model){
            parent::__construct($license,$driver); // asi se declara la SUPER CLASE
            $this->brand = $brand;
            $this->model = $model;
        }

        // GETTERS AND SETTERS
        public function getBrand(){
            return $brand;
        }
        public function setBrand($brand){
            $this->brand = $brand;
        }
        public function getModel(){
            return $model;
        }
        public function setModel($model){
            $this->model = $model;
        }

        // METODOS
        public function printDataCar(){
            echo "
                <br/>
                Licencia: {$this->getLicense()}<br/>
                Driver: {$this->getDriver()->getName()}<br/>
                Numero de pasajeros: {$this->getPassenger()}<br/>
                Marca: {$this->getBrand()}<br/>
                Modelo: {$this->getModel()}<br/>
            ";
        }
        
    }

?>